platforms/android/src/com/mb/android/MainActivity.java

```diff
@@ -63,17 +62,14 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 import org.apache.cordova.CordovaActivity;
 import com.mb.android.preferences.PreferencesProvider;
-import com.mb.android.webviews.CrosswalkWebView;
 import com.mb.android.webviews.IWebView;
```

Comment:
is it planned to get rid of `com.mb.*` things eventually?..

Reply:
Everything in `/platforms` should be removed from the repository eventually. Any custom functionality that is required should be in a plugin. (See #6)

Reply:
I like that new namespace :)